tests/python/contrib/test_ethosu/test_replace_conv2d.py

```diff
@@ -132,6 +132,28 @@ def _create_serial_conv2d_params(
     ]
 
 
+def get_conv2d_args(call, include_buffers=False, remove_constants=False):
+    """A method to extract the arguments from conv2d extern call."""
+    args = call.args
+    conv_args = []
+    remove_indices = [0]
+
+    if remove_constants:
+        remove_indices += [41, 42, 43, 44, 46, 47, 48, 49]
```

Comment:
docs : I think we need an explanation about these constants.

Reply:
Good point - I'll do it in the follow up!